implementations/python/sugar-lift-py-tests/tests/test_mutable_global_value.py

```
@@ -0,0 +1,78 @@
+    value = MutableGlobalValue(
+        "OPTIONS", "dict", binding.source_cid, binding.seal()
+    )
+    term = value.to_term(owner="test")
+
+    exits = value.subscript(StringValue("display.max_rows"), site)
+
+    assert isinstance(term, _Ctor)
+    assert binding.source_cid in repr(term)
+    assert binding.seal().cid in repr(term)
+    assert len(exits.exits) == 2
+    halted = next(exit for exit in exits.exits if isinstance(exit, Halted))
```

Ruff (0.16.0)

[error] Variable `exit` is shadowing a Python builtin

(A001)